docs/components/_index.scss

```diff
@@ -1,6 +1,7 @@
 /// load styles required to demonstrate each component
 @import "box/index";
 @import "grid/index";
+@import "panel/demo";
```

Comment:
Please rename file reference to `index`.

Reply:
Changes have been done, Please review